Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -190,6 +190,7 @@ void CtapAuthenticator::getAssertion()
     auto& options = std::get<PublicKeyCredentialRequestOptions>(requestData().options);
     auto internalUVAvailability = m_info.options().userVerificationAvailability();
     Vector<String> authenticatorSupportedExtensions;
+    CTAP_RELEASE_LOG("CtapAuthenticator::getAssertion uv: %d internalUvAvailability %d", options.userVerification, internalUVAvailability);
```

Comment:
The `CTAP_RELEASE_LOG` macro already inserts `CtapAuthenticator::`

Reply:
fixed and the one below also.